Kiosk/Bid Fulfillment/BidderNetworkModel.swift

```diff
@@ -1,12 +1,11 @@
 import Foundation
 import ReactiveCocoa
-import Swift_RAC_Macros
 import Moya
 
 class BidderNetworkModel: NSObject {
     // MARK: - Getters
 
-    var fulfillmentController: FulfillmentController
+    unowned let fulfillmentController: FulfillmentController
```

Comment:
I think even better might be not to inject the entire `FulfillmentController` in the first place, but instead have `FulfillmentController` inject functions that abstract over the queries / side effects the child wants. I've found it to be a really strong pattern is to never have a child component have an explicit reference to a parent. It also makes mocking much simpler -- just pass in a lamba with the assumed behavior.